### DISCUSSION TITLE
Docs or integration with Azure?

### DISCUSSION THREAD

**Discussion by devlife** [OrchardCMS/OrchardCore#11694] [Ideas]
I came across some [Microsoft Docs](https://docs.microsoft.com/en-us/azure/api-management/graphql-api) showing how to import a GraphQL API into Azure API Management. I wasn't sure if there were anything that you/we (the community) might be interested in doing here to integrate with this functionality. Maybe a new feature that accepts an APIM endpoint and whichever other settings and would automatically create/import/update an APIM endpoint?

I would imagine OC is used pretty broadly with Azure and thinking about how to better enable that story.

**Comment by Skrypt** [comment]
We already have GraphiQL to test Queries. What advantages does it give?

**Reply by devlife** [reply]
API Management is a service that offers a whole slew of tools for exposing APIs including (but definitely not limited to) DDoS protection, rate limiting, custom rules, custom mappings to different backend services. This wouldn't have anything to do with testing GraphQL queries so much as an integration with one of Azure's services. https://azure.microsoft.com/en-us/services/api-management/
